### PR TITLE
Skeleton

### PR DESCRIPTION
**Pull request by afonsobspinto**
- Add geppetto-hnn Skeleton
    - [x] Hello World
    - [x] Travis

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+os: linux
+sudo: false
+dist: trusty
+bundler_args: --retry 5
+node_js:
+  - "7"
+env:
+  global:
+    secure: dn0FPQ5IG4M/3kdwnyI78ElQ308Vc3QnKAvkWfwMFb8QxDqxQdnTo7AV1qTMtbLrDNkeEWIgi4nc7jmXNtvGTwOfhAULVh6606Qs5B+ezTdwzajbbFMI8SKQx/pnTojOMu8dx7V4lMoR/YWcojR0VC1IWVC62TGbSB1k5BDGgH0=
+git:
```
Comment (afonsobspinto) [inline .travis.yml]: @afonsobspinto Change this to use https

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    - libglapi-mesa
+    - libosmesa6
+before_script:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - export SLIMERJSLAUNCHER=/home/travis/firefox-58.0/firefox/firefox
+  - export LD_LIBRARY_PATH=/usr/lib/x86_64-linux-gnu/
+
+script:
+  - travis_wait travis_retry docker build -t="hnn-ui" github.com/MetaCell/HNN-UI#skeleton -f="./Dockerfile_dev"
```
Comment (afonsobspinto) [inline .travis.yml]: @afonsobspinto change skeleton to development after https://github.com/MetaCell/HNN-UI/pull/11
```suggestion
  - travis_wait travis_retry docker build -t="hnn-ui" github.com/MetaCell/HNN-UI#development -f="./Dockerfile_dev"
```

File: Utils.js [hunk 3]
```diff
@@ -0,0 +1,117 @@
+import {execPythonMessage, evalPythonMessage} from '../../js/communication/geppettoJupyter/GeppettoJupyterUtils';
```
Comment (tarelli) [inline Utils.js]: Are we using this file?

File: css/hnn.less [hunk 4]
```diff
@@ -0,0 +1,335 @@
...
+
+#mainContainer {
+  background-size: 100% 100%;
+  overflow: auto;
+  height: 100%;
+}
+#mainContainer > div {
+  height: 100% !important;
+}
+.actionButton {
```
Comment (tarelli) [inline css/hnn.less]: are we using these classes or do we know we will use them?

File: css/material.less [hunk 5]
```diff
@@ -0,0 +1,8 @@
+label {
+  margin-top: 0px;
+}
+
+input[type="text"], input[type="number"]{
```
Comment (tarelli) [inline css/material.less]: Are we using this class or do we know we will need it? i.e. let's double check it's needed and that we are not adding noise.

File: Utils.js (hunk 3 above)
Comment (afonsobspinto) [reply Utils.js]: 2 things to say here:

- 1st Iteration: I saw we were only using `execPythonMessage` `evalPythonMessage` so I brought those functions to ComponentsInitialization.js and deleted Utils.js

- 2nd Iteration: I was unable to find the method `evalPythonMessage`, and therefore removed the call to this method with apparent no side effects.

File: css/hnn.less (hunk 4 above)
Comment (afonsobspinto) [reply css/hnn.less]: I don't think I can answer that question, but I can tell you that I agree that is good practice to not add stuff that we think we might need in the future, so I removed everything besides the bare minimum, and will be adding the rest if/when needed.

File: css/material.less (hunk 5 above)
Comment (afonsobspinto) [reply css/material.less]: Idem

**Pull request merged**